lib/id3c/cli/io/pandas.py

```diff
@@ -77,8 +78,11 @@ def read_excel(io, sheet_name = 0, na_filter: bool = True):
     for these missing options could be added in the future with additional
     logic.
     """
+    warnings.filterwarnings(action='ignore', message='Data Validation extension is not supported and will be removed',
+        category=UserWarning, module='openpyxl', append=False)
+
```

Comment:
Moved. I originally had it at the top but thought it would make sense to put it right where it's used, but I see your point about it being called more than once.